tests/snuba/search/test_backend.py

```diff
@@ -3975,12 +3973,21 @@ def test_sort_user(self):
             date_from=self.base_datetime - timedelta(days=29),
         )
 
+    def test_sort_trends(self):
+        self.run_test(
+            "is:unresolved",
+            [self.group1, self.group2],
+            None,
+            sort_by="trends",
+            date_from=self.base_datetime - timedelta(days=30),
+        )
+
     def test_sort_priority(self):
         self.run_test(
             "is:unresolved",
             [self.group1, self.group2],
             None,
-            sort_by="priority",
+            sort_by="trends",
```

Comment:
Small thing here, this looks identical to the previous test? We also want to keep the tests for priority since the frontend will still be sending priority until the next PRs get merged